lib/index.ts

```diff
@@ -78,22 +78,17 @@ export function useServer(server: IMochaServer) {
 // mocha, and we use it too to start up a REPL when this option is used.
 const noexit: boolean = process.argv.includes("--no-exit") || process.argv.includes('-E');
 
-// Start up the webdriver and serve files that its browser will see.
-before(async function() {
-  this.timeout(20000);      // Set a longer default timeout.
-
+/**
+ * Create a driver, with all command-line options applied.  Extra options can be passed
```

Comment:
Are these options sufficient to customize userAgent? I think it would be helpful to include an example in this comment.